src/assets/js/shared/components/MusicTile.css

```diff
@@ -0,0 +1,17 @@
+.musicTile--container {
+  width: 50%;
+  height: 50%;
+  min-width: 150px;
+  min-height: 150px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.musicTile--songTitle {
```

Comment:
this is a new element so it should be `music-tile__song-title`... if the song title had changing colours then you would append it with `--` which is a modifier.. so like  `music-tile__song-title--blue` or ` `music-tile__song-title--green`